independent-projects/arc/runtime/src/main/java/io/quarkus/arc/InjectableBean.java

```diff
@@ -97,15 +97,15 @@ default boolean isNullable() {
 
     @Override
     default boolean isAlternative() {
-        return getAlternativePriority() != null;
+        return false;
```

Comment:
Why? The previous implementation (delegating to `getAlternativePriority`) seems still valid.

Reply:
Haha no it doesn't, that would result in an infinite regress. Please ignore.

Reply:
Well, the default impl of `getAlternativePriority()` now calls `isAlternative()`. Previously, we only had `alternativePriority` but now we have two separate attributes `alternative` and `priority`.

Reply:
Yea sure thing, my bad, I was distracted.